### PR TITLE
Expect the connection to be lost when using restart/stop service call buttons

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

When restart and stop service calls are executed, the websocket connection will stop, the service call was still successful so we should indicate that with actionSuccess and not actionError

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/buttons/ha-call-service-button.js [hunk 1]
```diff
@@ -69,10 +69,16 @@ class HaCallServiceButton extends EventsMixin(PolymerElement) {
           el.$.progress.actionSuccess();
           eventData.success = true;
         },
-        function () {
+        function (err) {
+          if (err?.error?.message && err.error.message === "Connection lost") {
```
Comment (bramkragten) [inline src/components/buttons/ha-call-service-button.js:73]: We should only do that for specific services, not for all?

Comment (ludeeus) [reply src/components/buttons/ha-call-service-button.js:73]: In which case would other services cause this?

Comment (bramkragten) [reply src/components/buttons/ha-call-service-button.js:73]: If the connection is lost? 😄

Comment (ludeeus) [reply src/components/buttons/ha-call-service-button.js:73]: vaccum.clean?

**Pull request closed without merging**